chord_metadata_service/mohpackets/utils.py

```diff
@@ -1,39 +1,17 @@
-import json
 import logging
-import re
 from enum import Enum
 
 logger = logging.getLogger(__name__)
 
 
-def get_schema_url():
+def get_schema_version():
     """
-    Retrieve the SHA URL from schema.json.
-    It reads the first 6 lines of the JSON file and extracts the URL from the "description".
+    Reads the version number from the schema file.
     """
-    url_pattern = r"https://[^\s]+"  # get everything after https
-
-    schema_url = "None"
-
-    try:
-        with open(
-            "chord_metadata_service/mohpackets/docs/schema.json", "r"
-        ) as json_file:
-            # Read the first 6 lines of the JSON file only
-            # The line we are looking for is on the 6th
-            first_6_lines = [next(json_file) for _ in range(6)]
-            first_6_lines.extend(["}", "}"])  # make valid JSON
-            # Concatenate the lines to form a JSON string
-            json_str = "".join(first_6_lines)
-            data = json.loads(json_str)
-            desc_str = data["info"]["description"]
-            schema_url = re.search(url_pattern, desc_str).group()
-    except Exception as e:
-        logger.debug(
-            f"An error occurred while fetching the schema URL. Details: {str(e)}"
-        )
-
-    return schema_url
+    file_path = "chord_metadata_service/mohpackets/docs/schema_version.txt"
+    with open(file_path, "r") as file:
+        version = file.read().strip()
+    return int(version)
```

Comment:
This should be a string, since it's a SHA and not an int.

Reply:
good catch!